### DISCUSSION TITLE
Help needed: IP not set after WiFi setup - Lanbon L8 - v0.7.0-rc12 (cd34d89)

### DISCUSSION THREAD

**Discussion by jscruz** [HASwitchPlate/openHASP#827] [Q&A] [answered]
Hello,

I'm using a Lanbon L8 and using the stable version 0.6.3 is working fine. When I tried to use version v0.7.0-rc12 (cd34d89) 2024-05-24 through the web installer, after setting up the WiFi, it seems the device is constantly asking for an IP to the DHCP server. At that point the device is not reachable.

I've tried to reflash the device and connect it to a different WiFi and it seems the behaviour is the same. No IP is assigned and then when I check the logs from my router, it seems the request for DHCP negotiation is happening every second.

I've tried restarting the device, the router, assigning a fixed IP for the MAC address through the router and it is still not working.

Any ideas why this is happening and how to solve it?

Thanks!

**Comment by jscruz** [comment, marked as answer]
I've solved it the hard way. I've forked the project and hardcoded the IP, compile it for Lanbon L8, write the flash and hey presto, it works.

**Reply by lameute76** [reply]
Hello, can you explain what you have done? , i have exactly the same problem !
